### DISCUSSION TITLE
Alternative to deprecated setImageClipMask() method???

### DISCUSSION THREAD

**Discussion by ragcsalo** [ImageMagick/ImageMagick#2989] [General]
Is there an alternative to the deprecated setImageClipMask() method?
I am using Imagick v7.0 on a new hosting server, and unfortunately the above method is completely missing, so I'm in a big trouble now... :-((((( Basically I'd like to blur only parts of an image, and would like to use a .PNG mask for this.

**Comment by snibgo** [comment]
As a guess, try SetImageMask().
